binderhub/tests/test_repoproviders.py

```diff
@@ -67,6 +68,20 @@ def test_spec_processing(spec, raw_user, raw_repo, raw_ref):
     assert repo_url == spec
 
 
+async def test_hydroshare():
```

Comment:
Suggest to add one more test based on a DOI, i.e. `10.4211/hs.b8f6eae9d89241cf8b5904033460af61`, as users are likely to "find" that first. See also comment above on favouring DOIs over direct URLs.

Reply:
@nuest.  I added a test using a DOI.  It looks like Travis failed with `Failed: JupyterHub should be available on Travis`.  I don't think this is related to my changes and maybe just a hiccup in the testing infrastructure.  Can we kick it off again?

Reply:
added the DOI test